insights/core/__init__.py

```diff
@@ -1542,3 +1542,109 @@ def __init__(self, *args, **kwargs):
         deprecated(AttributeDict, "Please set attributes explicitly.")
         super(AttributeDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+class ModInfo(CommandParser):
+    """
+    This will parase the inforamtion about a kernel module, the module
+    info will be stored in dictionary format.
+
+    Sample output:
+        filename:       /lib/modules/3.10.0-993.el7.x86_64/kernel/drivers/net/ethernet/intel/i40e/i40e.ko.xz
+        version:        2.3.2-k
+        license:        GPL
+        description:    Intel(R) Ethernet Connection XL710 Network Driver
+        author:         Intel Corporation, <e1000-devel@lists.sourceforge.net>
+        retpoline:      Y
+        rhelversion:    7.7
+        srcversion:     DC5C250666ADD8603966656
+        alias:          pci:v00008086d0000158Bsv*sd*bc*sc*i*
```

Comment:
Since the value of e.g. `alias`, `firmware`, is in the type of `list`, better explain it in the docstring and show an example in the `Examples`